src/shared/components/ToolBar/ToolBar.tsx

```diff
@@ -5,19 +5,29 @@ import { StyledButton, StyledToolBar, StyledContainer } from './styles';
 import logo from '../../data/logo.svg';
 
 
-const ToolBar: React.FC<{}> = (props) => {
+const ToolBar: React.FC<{pageType: string}> = (props) => {
   const navigate = useNavigate();
+  const pageType = props.pageType;
+  let homeColor: string = '#020166';
+  let favoritesColor: string = '#020166';
+
```

Comment:
Try use useMemo for the declaration of variable, relaying on other updating variable. try to avoid if/if else directly (only inside fucntions or hooks). Also, all constants like  #94D97E store in constants file for easier modification